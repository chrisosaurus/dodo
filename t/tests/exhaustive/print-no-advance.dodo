e/hello/
p

e/hello/
p5

e/hello/
p0

e/hello/

# why not?
w/ahoy!/

