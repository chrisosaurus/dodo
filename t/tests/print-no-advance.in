hello this is other stuff
