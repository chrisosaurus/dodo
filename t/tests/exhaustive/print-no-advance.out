ahoy! this is other stuff
